### ISSUE TITLE
Css IE exception

### ISSUE THREAD

**Issue by jbdelhommeau** [css/csso#214]
Hello,
In new bootstrap they are things like:

```css
  margin: 4px 0 0;
  margin-top: 1px \9; // IE8-9
```

After csso restult are:

```css
  margin: 1px 0 0;
```

The \9 value are ignored. But it's important :)

**Comment by twobyte** [comment]
Me too. Switched to gulp-minify-css – seems to work.

**Comment by lahmatiy** [comment]
Fixed by #166 
Coming in next release
